src/riscv_private.h

```diff
@@ -132,8 +140,11 @@ struct riscv_internal {
 #if !RV32_HAS(JIT)
     block_map_t block_map; /**< basic block map */
 #else
+    bool exit /**< Determine the main thread is exit or not */;
```

Comment:
Rename to `quit` in order to distinguish from the state of being exited from the execution or program being terminated. In this case, using `quit` is much clearer.